### PR TITLE
Add more zoom levels

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

**Comment by yvaucher** [comment]
@ybolognini @fredj not sure about this fix

File: base_geoengine/static/src/js/geoengine_view.js [hunk 1]
```diff
@@ -102,10 +102,10 @@ openerp.base_geoengine = function(openerp) {
...
                     break;
                 case "swisstopo":
-                    var resolutions = [4000, 3750, 3500, 3250, 3000, 2750, 2500, 2250, 2000, 1750, 1500, 1250, 1000, 750, 650, 500, 250, 100, 50, 20, 10, 5, 2.5];
-                    if (l.swisstopo_type == 'ch.swisstopo.swissimage') {
-                        resolutions.push(2, 1.5, 1, 0.5);
-                    }
+                var resolutions = [4000, 3750, 3500, 3250, 3000, 2750, 2500, 2250, 2000, 1750, 1500, 1250, 1000, 750, 650, 500, 250, 100, 50, 20, 10, 5, 2.5, 2, 1.5, 1];
+//                    if (l.swisstopo_type == 'ch.swisstopo.swissimage') {
+//                        resolutions.push(2, 1.5, 1, 0.5);
+//                    }
```
Comment (guewen) [inline base_geoengine/static/src/js/geoengine_view.js]: Couldn't the comment be removed?

Comment (yvaucher) [inline base_geoengine/static/src/js/geoengine_view.js]: That's why it is as WIP

**Comment by fredj** [comment]
Looks good to me

**Comment by yvaucher** [comment]
@fredj Can we remove those commented lines?

Don't we loose 0.5 resolution here?

**Comment by fredj** [comment]
The commented lines can be removed

**Comment by fredj** [comment]
@yvaucher good catch ! `0.5` must be added to the list

**Comment by yvaucher** [comment]
@guewen @fredj Fixed

**Comment by fredj** [comment]
+1

**Comment by yvaucher** [comment]
:+1:

**Comment by lmignon** [comment]
:+1:  even if I don't live in Swiss.. :smile:

**Comment by nbessi** [comment]
:+1:

**Pull request merged**